### PR TITLE
Add a container for curators to facilitate users running geo-importer

### PR DESCRIPTION
**Pull request by pmb59**
We would like to have a container/Docker image of a conda env so that other users can just use the image to run those scripts without struggling with installing all the dependencies.

### CODE REVIEW

**Comment by pmb59** [comment]
@YalanBi if installation of perl modules with cpanm is necessary perhaps we could use a `post_install_dockerfile` as described in the repo README

File: atlas-geo-importer/templated-conda-env.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: geo-importer
+channels:
+  - conda-forge
+  - ebi-gene-expression-group
+  - bioconda
+dependencies:
+  - magetab-curation-scripts
+  - perl-config-tiny
+  - perl-xml-parser
+  - perl-atlas-modules=0.3.1
```
Comment (anilthanki) [inline atlas-geo-importer/templated-conda-env.yaml:10]: I am struggling to test this package.. 

I keep on getting 
```Solving environment: failed

ResolvePackageNotFound: 
  - perl-atlas-modules=0.3.1
 ```

p.s. I am on Mac with M1 chip

Comment (pmb59) [reply atlas-geo-importer/templated-conda-env.yaml:10]: are you using the correct [channel](https://anaconda.org/ebi-gene-expression-group/perl-atlas-modules)?

Comment (anilthanki) [reply atlas-geo-importer/templated-conda-env.yaml:10]: I used the yaml file so should be using those channels...

There are three packages of ```perl-atlas-modules``` 
* version ```0.3.1``` in ```bioconda``` 
* version ```0.1.3``` and ```0.1.4``` in ```ebi-gene-expression-group```

**Review by anilthanki — APPROVED** [review]

**Pull request merged**